source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -77,9 +77,10 @@ AndroidContext: class extends OpenGLContext {
 		sourcePointer := eglImage buffer lock(false)
 		length := channels * eglImage size width * eglImage size height
 		buffer := ByteBuffer new(sourcePointer, length,
-			func (buffer: ByteBuffer) {
+			func (b: ByteBuffer) {
 				eglImage buffer unlock()
 				this recyclePacker(gpuRgba)
+				b forceFree()
```

Comment:
Because this closure defines what should happen when the ByteBuffer gets freed. I only want to recycle the backend memory and free the object data for the ByteBuffer. A call to `free` would result in an infinite loop.